script.js

```diff
@@ -171,7 +171,7 @@ const onMapLoad = async () => {
         mostRecentlyUpdatedAt: item.gsx$mostrecentlyupdated.$t
       }
       const location = _.pickBy(rawLocation, val => val != '')
-      const status = getStatus(item.gsx$color.$t)
+      const status = getStatus(item.gsx$color.$t) || {}
```

Comment:
This is a naive question but is there a better fallback e.g. an 'unknown status'? 
If not or that should be done as follow on will approve as is

Reply:
Yes. There's definitely a better solution. We can open a general data robustness and QA issue later. This is a penny in the circuit breaker.

Reply:
This is what I was working on:

```
      if (!status) {
        console.error("Malformed data for "+location.name+", could not find status: "+item.gsx$color.$t)
        return
      }
```

Reply:
love it, please open a pr